ai_diffusion/model.py

```diff
@@ -622,6 +632,12 @@ def _finish_job(self, job: Job, event: ClientEvent):
             self.jobs.notify_cancelled(job)
             self.progress = 0
 
+    def _apply_resize_command(self, cmd: ResizeCommand, job: Job):
+        try:
+            job.params.resize_canvas = bool(cmd.resize_canvas)
+        except Exception as e:
+            log.warning(f"Failed to store resize command from custom workflow: {e}")
+
```

Comment:
```suggestion
    def _apply_resize_command(self, cmd: ResizeCommand, job: Job):
        job.params.resize_canvas = cmd.resize_canvas

```

I think you can just put this 1 line above, instead of having a function